pbj-core/pbj-runtime/src/test/java/com/hedera/pbj/runtime/ProtoWriterToolsTest.java

```diff
@@ -192,6 +212,16 @@ void testWriteInteger_unsupported(FieldType type) {
         assertThrows(RuntimeException.class, () -> writeInteger(bufferedData, definition, RNG.nextInt()));
     }
 
+    @Test
+    void testWriteLong_Zero() {
```

Comment:
s/Zero/zero/ , or better use something like "default", cause the test case actually verifies how default values are (not) written.

Reply:
Changed to `zero`.